templates/charge.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+  <h2>Thanks, you paid <strong>{{amount}}</strong>!</h2>
```

Comment:
can you add screenshots to this PR showing how this template would appear? 

Also call it "donate-successful.html" or something like that, we  don't want to think of them as charges :)